### DISCUSSION TITLE
Use float with less precision for variables

### DISCUSSION THREAD

**Discussion by M1chV** [anyoptimization/pymoo#441] [Q&A] [answered]
Hi!
I have a large amount of **`n_var`** and I want to have float type of this variables and by default it's **`np.float64`**. I was trying to setup **`vtype=np.float16`** or **`vtype=np.float32`**, but I still have **`np.float64`**. 
So, do we have solution for it, or I need to create own type like [https://pymoo.org/customization/custom.html](https://pymoo.org/customization/custom.html)?
Thanks :)

**Comment by blankjul** [comment, marked as answer]
Currently, this is not really supported. Most operators use `float` which I think for most hardware maps to `float64`. However, I should also mention that no algorithm in pymoo is suitable for truly large-scale problems (n_var > 10,000 or so). There are EA variants that have been proposed to work more efficient though.
